mlir/lib/Dialect/SCF/Transforms/TileUsingInterface.cpp

```diff
@@ -1954,18 +2024,23 @@ getUntiledConsumerFromSlice(RewriterBase &rewriter, Operation *sliceOp) {
 /// Implementation of fusing consumer of a single slice by computing the
 /// slice of the consumer in-place for scf loop.
 FailureOr<scf::SCFFuseConsumerOfSliceResult>
-mlir::scf::tileAndFuseConsumerOfSlice(RewriterBase &rewriter,
-                                      Operation *candidateSliceOp) {
+mlir::scf::tileAndFuseConsumerOfSlice(
+    RewriterBase &rewriter, Operation *candidateSliceOp,
+    MutableArrayRef<LoopLikeOpInterface> loops) {
```

Comment:
If we're doing this, can we update `loops` with the newly created loops by the fusion?

Reply:
That is what happens now.

Reply:
Oh it's hidden inside `addInitOperandsToLoopNest`, didn't catch that. Makes sense then.